settings.php

```diff
@@ -38,6 +38,13 @@ function ecs($configoption, $params = null) {
     $settings->add(new admin_setting_configtext('equella_url', ecs('url.title'), ecs('url.desc'), ''));
     $settings->add(new admin_setting_configtext('equella_action', ecs('action.title'), ecs('action.desc'), ''));
 
+    $userfieldoptions = array();
+    $userfieldoptions = array('default' => ecs('userfield.username'));
```

Comment:
My reviewer at this time thought it was better. Also it might avoid any problem while looking for the profile field. (I think to avoid duplicate ? )
Honestly I'm not too sure now.
But looks it has been working for one of our site for a year now.